character-details.mjs

```diff
@@ -38,41 +38,42 @@ character-details-page .character-details p span {
 `;
 document.body.appendChild(styles);
 
-export default Component.extend({
-  tag: "character-details-page",
+export default class CharacterDetailsPage extends StacheElement {
+	static view = `
+		<div class="breadcrumbs">
+			<div>
+				<a href="{{ routeUrl(page="list" query=this.query)}}" class="search">&lt; Characters</a>
```

Comment:
Should there be a space before the closing `}}`? I'm not sure what the consensus was, but it looks like this is what you have everywhere else.